### ISSUE TITLE
[Bug]: warszawa19115.pl cant pull data from API

### ISSUE THREAD

**Issue by krystjanik** [mampfes/hacs_waste_collection_schedule#2966]
### I Have A Problem With:

A specific source

### What's Your Problem

When trying configure addon on my location (which has been working flawlessly for a long time) got error:
"The source returned an invalid response: "object of type 'NoneType' has no len()". Please check the provided arguments and try again."

### Source (if relevant)

warszawa19115_pl

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [ ] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [ ] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by MarcinGmurczyk** [comment]
Same here
```This error originated from a custom integration.

Logger: custom_components.waste_collection_schedule.waste_collection_schedule.source_shell
Source: custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py:160
integration: Waste Collection Schedule (documentation)
First occurred: 03:06:23 (2 occurrences)
Last logged: 11:18:07

fetch failed for source Warsaw: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 158, in fetch entries: Iterable[Collection] = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/warszawa19115_pl.py", line 127, in fetch or len(calendar_result[0]["harmonogramy"]) <= 0 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ TypeError: object of type 'NoneType' has no len()`

**Comment by miedzeszyn** [comment]
And same here.
I suspect some API changed as ZTM transport also stop working few days ago.

**Comment by m-wyz** [comment]
+1

**Comment by m-wyz** [comment]
Already solved:
https://github.com/mampfes/hacs_waste_collection_schedule/pull/2971

**Comment by 5ila5** [comment]
fixed in #2971

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>
